typescript/007-rna-transcription/rna-transcription.ts

```diff
@@ -0,0 +1,15 @@
+export function toRna(dnaSequence: string): string {
+  const transcriptionMap = new Map<string, string>([
+    ['G', 'C'],
+    ['C', 'G'],
+    ['T', 'A'],
+    ['A', 'U'],
+  ]);
+
+  return dnaSequence.split('').map(nucleotide => {
+    if (!transcriptionMap.get(nucleotide)) {
+      throw new Error(`Invalid input DNA.`);
+    }
+    return transcriptionMap.get(nucleotide);
```

Comment:
Nit: `get` is called twice in the happy path. However, this is a very minor overhead so it's fine.